types/moment-timezone/index.d.ts

```diff
@@ -55,7 +54,15 @@ declare module "moment" {
         names(): string[];
         guess(ignoreCache?: boolean): string;
 
-        setDefault(timezone?: string): MomentTimezone;
+        setDefault(timezone: string): MomentTimezone;
+
+        pack(data: object): string;
+
+        packBase60(x: any, y: number): any;
```

Comment:
Is there a reason why the `x` parameter is `any` type? I believe it should be `number`. Parameter `y` is optional.

Also, doesn't this function returns a `string`?